### PR TITLE
Update components

### PR DESCRIPTION
**Pull request by ymann**
Update to tracker component - updated logic so that targets appear in correct positions and data from Tracker_Init endpoint is stored and used.

Update to heatmap component - fixed flipped y-axis and issue with smoothing

added Windows for ReadMe

### CODE REVIEW

File: liveviz/src/components/heatmap/HeatmapUtils.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const drawHeatMap = (canvas, data, {min, max}) => {
     for (var j = 0; j < n; j++) {
       const val = scale(data[i][j], min, max, 0, 255);
       ctx.fillStyle = rainbowColor(val);
-      ctx.fillRect(i * scaleX, height - (j+1) * scaleY, scaleX, scaleY);
+      ctx.fillRect(i * scaleX - 1, j * scaleY - 1, scaleX + 1, scaleY + 1);
```
Comment (viebel) [inline liveviz/src/components/heatmap/HeatmapUtils.js]: please add a comment explaining why we need the `+1`

File: liveviz/src/components/trackermap/ui/TrackerUI.js [hunk 2]
```diff
@@ -71,15 +71,19 @@ class TrackerUI extends Component {
     this.state = {
       width: 0,
       height: 0,
+      arenaWidth: this.props.room.Data[0][0],
```
Comment (viebel) [inline liveviz/src/components/trackermap/ui/TrackerUI.js]: The arena dimensions and sensor positions should not be on the state. They should be retrieved in the `render` function.

Comment (ymann) [inline liveviz/src/components/trackermap/ui/TrackerUI.js]: Fixed in next commit

File: liveviz/src/components/heatmap/HeatmapUtils.js (hunk 1 above)
Comment (ymann) [inline liveviz/src/components/heatmap/HeatmapUtils.js]: Fixed in next commit

File: liveviz/src/components/trackermap/ui/TrackerUI.js [hunk 3]
```diff
@@ -75,11 +75,15 @@ class TrackerUI extends Component {
...
   componentDidMount() {
     const {width, height} = this.domElement.getBoundingClientRect();
+
     this.setState(merge(this.state, {width, height}));
   }
   render() {
-    const {width, height} = this.state,
-    [arenaWidth, arenaHeight] = [20, 10]
+    const {width, height} = this.state;
+    const arenaWidth: this.props.room.Data[0][0]
```
Comment (viebel) [inline liveviz/src/components/trackermap/ui/TrackerUI.js]: have you tested it? 
`const arenaWidth: this.props.room.Data[0][0]` is not js valid syntax!!!

Try something like:

`const {arenaWidth, arenaHeight} =  this.props.room.Data[0]`

File: liveviz/src/components/trackermap/ui/TrackerUI.js [hunk 4]
```diff
@@ -75,11 +75,15 @@ class TrackerUI extends Component {
...
   componentDidMount() {
     const {width, height} = this.domElement.getBoundingClientRect();
+
     this.setState(merge(this.state, {width, height}));
   }
   render() {
-    const {width, height} = this.state,
-    [arenaWidth, arenaHeight] = [20, 10]
+    const {width, height} = this.state;
+    const arenaWidth = this.props.room.Data[0][0]
```
Comment (viebel) [inline liveviz/src/components/trackermap/ui/TrackerUI.js]: Use ES6 destructuring syntax:

```
const {arenaWidth, arenaHeight} = this.props.room.Data[0]
same for sensorX and sensorY
```

**Pull request merged**